fragdenstaat_de/fds_cms/templates/foirequest/cms_plugins/_map.html

```diff
@@ -0,0 +1,30 @@
+{% load i18n %}
+{% with plugin_id=instance.id|stringformat:"i" %}
+    {% with data_id="foirequest_map_plugin_data_"|add:plugin_id %}
+        {{ mapdata|json_script:data_id }}
+        <template id="foirequest_map_plugin_template_{{ instance.id }}">
+            <div class="foirequest-legend">
+                <b>{% trans "Status" %}</b>
+                <br>
+                <i style="background-color:#ffc107"></i> {% trans "Pending" %}
```

Comment:
should not only be color, but another visual element (e.g. shape) https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html